spec/jobs/form526_confirmation_email_job_spec.rb

```diff
@@ -0,0 +1,66 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+require 'notifications/client'
+
+RSpec.describe Form526ConfirmationEmailJob, type: :worker do
+  before { Sidekiq::Worker.clear_all }
+
+  describe '#perform' do
+    let(:notification_client) { double('Notifications::Client') }
+
+    context 'defaults' do
```

Comment:
Context blocks should use "with $CONTEXT" or "when $CONTEXT" so the spec block reads more naturally:

'with defaults' it 'sends a confirmation email'
or -

'when user is unauthorized' it 'does not send an email' etc.

Reply:
Good to know! Thanks for the suggestion, we've approved it